src/shared/components/publications-and-talks-section/videos-list/index.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import ReactPlayer from 'react-player';
+import PropTypes from 'prop-types';
+
+import RemainingVideo from 'shared/components/publications-and-talks-section/videos-list/remaining-video';
+import styles from './index.module.css';
+
+class PublicationsAndTalksList extends Component {
+    constructor(props) {
+        super(props);
+        const { videosArr } = this.props;
+        const remainingVideos = [];
```

Comment:
You don't need this array, nor having all the videos in the state.

It's enough to have the index of the active video in the state.